docs/calamari/Intro.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_position: 1
+id: Intro
+title: Intro
+---
+
+Calamari, Manta Network's canary net, is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps.
+
+## Calamari Roadmap
+
+| Kusama Parachain Auction         | On-Chain Governance                                    | General Calamari Tasks                      |
+| -------------------------------- | ------------------------------------------------------ | ------------------------------------------- |
+| Release Calamari PLO Strategy ✅ | Community Governance Forum                             | Calamari Web Front-End Release              |
+| Publish Chain Genesis 🚧         | Technical Council Formation                            | Calamari Chain Launch 🚧                    |
+| Kusama Parachain Auction ✅      | Runtime upgrade (Remove Sudo / Add Governance Pallets) | MariPay Testnet (Testnet V2)                |
+| Win a Parachain Slot ✅          | KMA Token Launch 🚧                                    | Zero-Knowledge Proof Circuit Audit          |
```

Comment:
@AlbionaHoti @stechu the `On-Chain Governance` stuff seems lacking.
We should at least mention the Council chamber.
Furthermore we can a little more detail like here -> https://polkadot.network/launch-roadmap/#:~:text=Governance%20and%20Removing,Learn%20More

Reply:
@ghzlatarev The notes are taken from this document: https://emphasized-seed-161.notion.site/3b1b61e0aee8484396d674f4653e0813. I'll write on our channel now with some questions.